source/back.tex

```diff
@@ -61,20 +61,20 @@
   Prentice-Hall, ISBN 0-13-117003-1, copyright \copyright 1995 P.\,J.\ Plauger
 \bibitem{linalg-stable}
   J.\ Demmel, I.\ Dumitriu, and O.\ Holtz,
-  \doccite{Fast linear algebra is stable},
-  Numerische Mathematik 108 (59--91), 2007.
+  Fast linear algebra is stable,
+  \doccite{Numerische Mathematik}, Vol.\ 108, No.\ 1, pp.\ 59--91, 2007.
```

Comment:
We seem to \doccite the title of the book / article elsewhere in this list, not (necessarily) the journal name.

Agreed with the rest of the adjustments.